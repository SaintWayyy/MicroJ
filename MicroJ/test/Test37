/* Test airthmetic operator */
int main(){
    /* Addition */
    print(4 + 3);   /* The result should be 7. */
    print(12.6 + 9.84); /* The result should be 22.440000. */

    /* Subtraction*/
    print(4 - 3);   /* The result should be 1. */
    print(12.6 - 9.84);    /* The result should be 2.760000. */

    /* Multiplication*/
    print(4 * 3);  /* The result should be 12. */
    print(12.6 * 9.84);    /* The result should be 123.984000 */

    /*Division*/
    /* Integer division will remove the trailing numerals to 0, 
    for example 4 / 3 is 1. */
    print(4 / 3);   /* The result should be 1 */
    print(12.6 / 9.84);   /* The result should be 1.2804878049 */
}