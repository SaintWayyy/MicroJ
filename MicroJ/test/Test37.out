7
22.440000
1
2.760000
12
123.984000
1
1.280488
